app/controllers/recipes_controller.rb

```diff
@@ -28,9 +28,9 @@ def create
   end
 
   def edit
-    
+
```

Comment:
best practice in PR's to change as little files as possible when it has nothing to do with your task. I would recommend that you revert this if it doesn't concern this task.

Reply:
Thanks for the feedback we'll have more attention in the next story!